.changeset/eight-trainers-deny.md

```diff
@@ -0,0 +1,4 @@
+---
+---
```

Comment:
```suggestion
"@fuel-ts/account": patch
"@fuel-ts/program": patch
---
```

Reply:
@maschad @nedsalk Does this mean there's still work to be done regarding this changeset automation?

Reply:
Same [here](https://github.com/FuelLabs/fuels-ts/pull/2767#discussion_r1677397448).

Reply:
And [here](https://github.com/FuelLabs/fuels-ts/pull/2766#discussion_r1677396974), and probably more places.

Reply:
Relates to:
- https://github.com/FuelLabs/fuels-ts/issues/2733